lib/guardian_db.ex

```diff
@@ -120,4 +135,17 @@ defmodule GuardianDb do
   end
 
   def repo, do: @repo
+
+  defp token_types do
+    Application.get_env(:guardian_db, GuardianDb, [])
```

Comment:
Even though `Application.get_env` is just an ETS lookup, I agree it would be better to pull that call into a module attribute. It did make testing easier by allowing the config options to change at runtime, but that was definitely not a good reason to keep that implementation. I'll fix that and pull the new tests into a separate file to run from a separately compiled mix task. That should also keep tests cleaner from the default config options.

I can also remove the `to_string/1` calls and assume that `@token_types` is a list of atoms. The only time I see the need for a conversion is with `on_verify/2` where `claims["typ"]` will be a string. What would be the recommended method for comparing this string to the list of atoms? I am hesitant to pull in `String.to_existing_atom/1`.

Good call out on the pipe styling. I should have learned by now after all the time Credo yells at me about it :D